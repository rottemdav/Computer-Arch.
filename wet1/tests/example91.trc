16 5 18 1 local_history global_tables not_using_share
0x671005c N 0xab787c
0x36b0 N 0xb033fb88
0x3523714 N 0x41b4
0x14b4 T 0xb50
0x36b0 T 0xb033fb88
0xea7c T 0xa07e4
0xb31dafc N 0x441748
0x705fc T 0x1a83650
0xf503b2d0 T 0xc9a97818
0xb31dafc T 0x441748
0x705fc N 0x1a83650
0x788967c T 0xab0
0x5c76fc T 0x465c08
0x93c0 N 0xffde4
0x5c76fc N 0x465c08
0xf24 N 0x45c
0x6234 N 0xa44c8
0xea7c T 0xa07e4
0x705fc T 0x1a83650
0x75f2f8 T 0x9b988c
0xcfc N 0x613c4f3c
0x705fc T 0x1a83650
0x8596ca8 N 0x5d7c624
0x76598e8 T 0x83b7d8
0x4d390 N 0x38d0
0x671005c T 0xab787c
0xf328 T 0x774b038
0xcfc N 0x613c4f3c
0x4d8 T 0x7a0dd0
0x40b0ec N 0x2c8313c8
0xcfc N 0x613c4f3c
0x9108 T 0x8d803b5c
0x5c76fc N 0x465c08
0xac4 T 0xd2f5ec0
0xf2eb8 T 0x13200c
0x60e48 T 0x01c
0x6ab2ec20 T 0xbfb308
0x60e48 N 0x01c
0xf328 T 0x774b038
0xde50cd88 T 0xcfb8
0x78034 T 0x0590b0c
0xea7c N 0xa07e4
0xa7dec8ac N 0xac8a6254
0x3e0 T 0x1a14
0x32da8ec4 T 0xa3213c8
0x3523714 N 0x41b4
0x788967c N 0xab0
0x6234 T 0xa44c8
0x76598e8 T 0x83b7d8
0x78034 T 0x0590b0c
0xe11fc81c T 0x7bd87e0
0x78034 T 0x0590b0c
0x6234 N 0xa44c8
0x80e5a4 N 0xc0c
0xe11fc81c T 0x7bd87e0
0x6ab2ec20 N 0xbfb308
0x6ab2ec20 N 0xbfb308
0x5c76fc T 0x465c08
0x26ebd64 T 0x5c18
0x6ab2ec20 T 0xbfb308
0xf503b2d0 N 0xc9a97818
0xf24 T 0x45c
0x5c76fc T 0x465c08
0x78034 N 0x0590b0c
0x78034 N 0x0590b0c
0xf328 T 0x774b038
0x4afc N 0x7c4cac
0x6234 N 0xa44c8
0x6ab2ec20 T 0xbfb308
0xf328 T 0x774b038
0x8596ca8 N 0x5d7c624
0x3e0 N 0x1a14
0x60e48 N 0x01c
0x705fc T 0x1a83650
0xac4 T 0xd2f5ec0
0x2a100ec N 0xb4d4324c
0x36b0 T 0xb033fb88
0x26ebd64 T 0x5c18
0xf2eb8 N 0x13200c
0xf328 N 0x774b038
0xf503b2d0 T 0xc9a97818
0x4d390 N 0x38d0
0xd7cf8 T 0xddf0324
0x3e0 T 0x1a14
0x3e0 N 0x1a14
0xf503b2d0 N 0xc9a97818
0x8596ca8 N 0x5d7c624
0x26ebd64 N 0x5c18
0xac4 N 0xd2f5ec0
0x93c0 N 0xffde4
0x6ab2ec20 N 0xbfb308
0x234 T 0x5474
0x4d390 N 0x38d0
0xf24 N 0x45c
0x705fc N 0x1a83650
0x40b0ec T 0x2c8313c8
0xf328 N 0x774b038
0x6ab2ec20 N 0xbfb308
0x08c T 0x540aa98
0x8596ca8 N 0x5d7c624
0x788967c N 0xab0
0x4d8 T 0x7a0dd0
0x32da8ec4 T 0xa3213c8
0xe11fc81c T 0x7bd87e0
0x26ebd64 N 0x5c18
0x26ebd64 T 0x5c18
0xf2eb8 N 0x13200c
0xf24 N 0x45c
0x5c76fc N 0x465c08
0xde50cd88 N 0xcfb8
0x75f2f8 T 0x9b988c