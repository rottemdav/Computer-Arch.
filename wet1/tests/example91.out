0x671005c N 0x6710060
0x36b0 N 0x36b4
0x3523714 N 0x3523718
0x14b4 N 0x14b8
0x36b0 N 0x36b4
0xea7c N 0xea80
0xb31dafc N 0xb31db00
0x705fc N 0x70600
0xf503b2d0 N 0xf503b2d4
0xb31dafc N 0xb31db00
0x705fc N 0x70600
0x788967c N 0x7889680
0x5c76fc N 0x5c7700
0x93c0 N 0x93c4
0x5c76fc N 0x5c7700
0xf24 N 0xf28
0x6234 N 0x6238
0xea7c N 0xea80
0x705fc N 0x70600
0x75f2f8 N 0x75f2fc
0xcfc N 0xd00
0x705fc N 0x70600
0x8596ca8 N 0x8596cac
0x76598e8 N 0x76598ec
0x4d390 N 0x4d394
0x671005c T 0xab787c
0xf328 N 0xf32c
0xcfc N 0xd00
0x4d8 N 0x4dc
0x40b0ec N 0x40b0f0
0xcfc T 0x613c4f3c
0x9108 N 0x910c
0x5c76fc N 0x5c7700
0xac4 N 0xac8
0xf2eb8 N 0xf2ebc
0x60e48 N 0x60e4c
0x6ab2ec20 N 0x6ab2ec24
0x60e48 N 0x60e4c
0xf328 N 0xf32c
0xde50cd88 N 0xde50cd8c
0x78034 N 0x78038
0xea7c N 0xea80
0xa7dec8ac N 0xa7dec8b0
0x3e0 N 0x3e4
0x32da8ec4 N 0x32da8ec8
0x3523714 T 0x41b4
0x788967c N 0x7889680
0x6234 N 0x6238
0x76598e8 N 0x76598ec
0x78034 N 0x78038
0xe11fc81c N 0xe11fc820
0x78034 N 0x78038
0x6234 N 0x6238
0x80e5a4 N 0x80e5a8
0xe11fc81c T 0x7bd87e0
0x6ab2ec20 N 0x6ab2ec24
0x6ab2ec20 N 0x6ab2ec24
0x5c76fc N 0x5c7700
0x26ebd64 N 0x26ebd68
0x6ab2ec20 T 0xbfb308
0xf503b2d0 N 0xf503b2d4
0xf24 N 0xf28
0x5c76fc T 0x465c08
0x78034 N 0x78038
0x78034 T 0x590b0c
0xf328 N 0xf32c
0x4afc N 0x4b00
0x6234 N 0x6238
0x6ab2ec20 T 0xbfb308
0xf328 T 0x774b038
0x8596ca8 N 0x8596cac
0x3e0 N 0x3e4
0x60e48 N 0x60e4c
0x705fc N 0x70600
0xac4 N 0xac8
0x2a100ec N 0x2a100f0
0x36b0 T 0xb033fb88
0x26ebd64 N 0x26ebd68
0xf2eb8 T 0x13200c
0xf328 N 0xf32c
0xf503b2d0 N 0xf503b2d4
0x4d390 N 0x4d394
0xd7cf8 N 0xd7cfc
0x3e0 T 0x1a14
0x3e0 T 0x1a14
0xf503b2d0 N 0xf503b2d4
0x8596ca8 N 0x8596cac
0x26ebd64 N 0x26ebd68
0xac4 N 0xac8
0x93c0 N 0x93c4
0x6ab2ec20 N 0x6ab2ec24
0x234 N 0x238
0x4d390 N 0x4d394
0xf24 N 0xf28
0x705fc N 0x70600
0x40b0ec N 0x40b0f0
0xf328 N 0xf32c
0x6ab2ec20 N 0x6ab2ec24
0x8c N 0x90
0x8596ca8 N 0x8596cac
0x788967c N 0x7889680
0x4d8 N 0x4dc
0x32da8ec4 N 0x32da8ec8
0xe11fc81c N 0xe11fc820
0x26ebd64 N 0x26ebd68
0x26ebd64 N 0x26ebd68
0xf2eb8 N 0xf2ebc
0xf24 N 0xf28
0x5c76fc N 0x5c7700
0xde50cd88 N 0xde50cd8c
0x75f2f8 N 0x75f2fc
flush_num: 52, br_num: 111, size: 928b
